packages/palette/src/elements/Checkbox/Checkbox.story.tsx

```diff
@@ -32,7 +43,22 @@ export const Demo = () => {
         action("onClick")(selected)
       }}
     >
-      <Text>click me</Text>
+      <Text lineHeight="solid">
+        use a `solid` line-height to ensure vertical centering
```

Comment:
I see in force some usage of `pt="1px"` in the label because the default line-height/font-size combo is taller than the checkbox label; using a `solid` line-height is the appropriate fix there; so just pointing it out since it's probably common.